src/network/networkthread.py

```
@@ -15,8 +16,12 @@ def __init__(self):
         logger.info("init asyncore thread")
 
     def run(self):
-        while not self._stopped and state.shutdown == 0:
-            BMConnectionPool().loop()
+        try:
+            while not self._stopped and state.shutdown == 0:
+                BMConnectionPool().loop()
+        except Exception as e:
+            excQueue.put((self.name, e))
+            raise
```

If I remove `raise`, any exception will be ignored.